build.sbt

```diff
@@ -261,7 +261,10 @@ lazy val zincPersist = (projectMatrix in internalPath / "zinc-persist")
   .dependsOn(zincCore, zincCompileCore, zincCore % "test->test")
   .settings(
     name := "zinc Persist",
-    libraryDependencies += sbinary,
+    libraryDependencies ++= Seq(
+      sbinary,
+      snappy,
```

Comment:
Given the wide use of Zinc, I would rather avoid introducing compression library maintained by a single person (e.g. https://en.wikipedia.org/wiki/XZ_Utils_backdoor). The author is a long-time community member and this isn't meant to be a FUD, but generally I think we should be careful about introducing [native code](https://github.com/xerial/snappy-java/blob/master/src/main/resources/org/xerial/snappy/native/Mac/aarch64/libsnappyjava.dylib) that we didn't write unless it's heavily audited.

Reply:
Makes sense. I shall do more research on what other compression libraries are available. Hopefully there's one that we can better trust.